### PR TITLE
Replace "Twitter" logo with "X" logo on Community page

### PR DESCRIPTION
**Pull request by chansuke**
Fixes 43206
Update the icon and name of Twitter to X(Twitter)

### CODE REVIEW

**Comment by sftim** [comment]
Please look at https://deploy-preview-43583--kubernetes-io-main-staging.netlify.app/community/ 

That page still shows Twitter's old icon.

File: hugo.toml [hunk 1]
```diff
@@ -249,10 +249,10 @@ no = 'Sorry to hear that. Please <a href="https://github.com/USERNAME/REPOSITORY
 	desc = "Discussion and help from your fellow users"
 
 [[params.links.user]]
-	name = "Twitter"
+	name = "X(Twitter)"
 	url = "https://twitter.com/kubernetesio"
-	icon = "fab fa-twitter"
-	desc = "Follow us on Twitter to get the latest news!"
+	icon = "fab fa-x-twitter"
+	desc = "Follow us on X(Twitter) to get the latest news!"
```
[the remark this answers is not in the document]
Comment (chansuke) [reply hugo.toml]: @sftim Thank you so much for your review. I've update the description.

**Comment by sftim** [comment]
This is a fix for issue https://github.com/kubernetes/website/issues/43206

/retitle Replace "Twitter" logo with "X" logo on Community page

**Comment by Ritikaa96** [comment]
Hi @chansuke Please rebase your changes.

**Pull request merged**